cypress/support/step_definitions/getterSetterCalls.js

```diff
@@ -16,25 +16,509 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 import { Given } from '@badeball/cypress-cucumber-preprocessor';
+import UTILS, { fireLog } from '../cypress-support/src/utils';
 const CONSTANTS = require('../constants/constants');
 
 /**
- * @module settersGetterCalls
+ * @module getterSetterCalls
  * @function Given we test the '(.+)' getters and setters
  * @description Define and cache the fireboltCall object to use for getters and setters scenario by saving the object in env variable.
  * @param {String} key - key name of the fireboltCall setter/getter data.
  * @example
  * Given we test the 'CLOSED_CAPTIONS' getters and setters
+ * Given we test the 'CLOSED_CAPTIONS' getters and setters 'enabled' to 'true'
  */
-Given(/we test the '(.+)' getters and setters$/, async (key) => {
-  // Clear any current env.runtime variables
-  Cypress.env(CONSTANTS.RUNTIME, {});
-
-  // Look for the firebolt call key in firebolt data
-  cy.getFireboltData(key).then((parsedData) => {
-    // getFireboltData handles the case where the key is not found
-    // Save the object as env.runtime.fireboltCall
-    const runtime = { fireboltCall: parsedData };
-    Cypress.env(CONSTANTS.RUNTIME, runtime);
-  });
+Given(
+  /we test the '(.+)' getters and setters(?: '(.+)' to '(.+)')?$/,
+  async (key, attribute, value) => {
+    // Clear any current env.runtime variables
+    Cypress.env(CONSTANTS.RUNTIME, {});
+
+    // Look for the firebolt call key in firebolt data
+    cy.getFireboltData(key).then((parsedData) => {
+      // getFireboltData handles the case where the key is not found
+      // Save the object as env.runtime.fireboltCall
+      const runtime = { fireboltCall: parsedData };
+      if (attribute && value) {
```

Comment:
So in conjunction with the comment on the null scenarios from the feature file, do we want the ability to just provide `attribute`?

Reply:
Yes, we have some scenarios where we passing null value for set calls. I think it should be better if we have that support.
@neeradanelxsi